cli/src/commands/extensions/api.rs

```diff
@@ -256,24 +256,38 @@ async fn get_projects(
     api.get_projects(group.as_deref()).await.map_err(Error::from)
 }
 
+#[derive(Serialize)]
+struct CreatedProject {
+    id: ProjectId,
+    status: CreatedProjectStatus,
+}
+
+#[derive(Serialize)]
+enum CreatedProjectStatus {
+    Created,
+    Existing,
```

Comment:
I'm not sure "Existing" is the best choice here, but I don't have a better idea. Feels kinda weird when put next to "Created".

Maybe "Exists" would be better?